node/hyperbahn/index.js

```diff
@@ -304,10 +304,6 @@ function advertise(opts) {
             },
             self.advertisementTimeoutTime
         );
-    } else {
-        self.logger.info('HyperbahnClient advertisement timeout disabled', {
```

Comment:
put the log back, but move it to constructor, so it only log once, but not per advertise call.